src/epic.actions.js

```diff
@@ -16,7 +16,12 @@ async function getEpic({ id }) {
 
 async function describeEpic({ id }) {
```

Comment:
Can we add JSDoc comments to these functions?

Reply:
I'll add them to the functions I modified